libafl_bolts/src/tuples.rs

```diff
@@ -209,6 +209,49 @@ where
     }
 }
 
+/// Borrow each member of the tuple
+pub trait SplitBorrow<'a> {
```

Comment:
Is there a fancy functional programming name for "give me head and tail" that would fit here? Else this name is fine too

Reply:
it is split borrow for normal rust tuples, so I kept the name